core/unit_test/CMakeLists.txt

```diff
@@ -554,13 +554,12 @@ KOKKOS_ADD_ADVANCED_TEST( UnitTest_PushFinalizeHook_terminate
   endif()
   if(KOKKOS_ENABLE_LIBDL)
 
-    add_library(
+    KOKKOS_ADD_TEST_LIBRARY(
       kokkosprinter-tool SHARED
-      tools/printing-tool.cpp
+      SOURCES tools/printing-tool.cpp
+      DEPLIBS kokkoscore
```

Comment:
What does DEPLIBS mean here? It doesn't depend on kokkoscore

Reply:
I can remove it.

Reply:
But, yeah, it's a way to make test libraries depend on kokkoscore, etc